src/components/DayPickerNavigation.jsx

```diff
@@ -126,7 +126,7 @@ function DayPickerNavigation({
       {!isVerticalScrollable && (
         <div
           role="button"
-          tabIndex="0"
+          tabIndex={isDefaultNavPrev ? '0' : null}
```

Comment:
does `null` work, or does it have to be `undefined`?

Reply:
I wasn't able to find anything very helpful on google about what the value should be, so to be safe, I think I'll go with [this stack overflow ](https://stackoverflow.com/questions/36523225/dynamic-tabindex-attribute-in-jsx-react)solution